apps/mull-api/src/app/entities/location.entity.ts

```diff
@@ -9,7 +9,7 @@ export class Location implements ILocation {
   id: number;
 
   @Column('point')
-  point: string;
+  title: string;
```

Comment:
So this was changed because now title will be the location in common language and coordinates which will contain the coordinates if the user uses current location, otherwise the location's placeId will be used, i left an explanation below as to why we're using placeId instead of [Lat, Long]

Reply:
As discussed, the coordinate field will be added in the entity in #124